app/src/main/java/co/omisego/omgshop/extensions/APIErrorExt.kt

```diff
@@ -9,4 +9,7 @@ import co.omisego.omisego.model.APIError
  * Created by Phuchit Sirimongkolsathien on 20/6/2018 AD.
  * Copyright © 2017-2018 OmiseGO. All rights reserved.
  */
-fun APIError.isAuthError() = this.code == ErrorCode.CLIENT_INVALID_API_KEY || this.code == ErrorCode.USER_AUTH_TOKEN_NOT_FOUND
+
+fun APIError.isAuthError() = this.code == ErrorCode.CLIENT_INVALID_API_KEY ||
```

Comment:
`code in arrayOf(ErrorCode.CLIENT_INVALID_API_KEY, ErrorCode.USER_AUTH_TOKEN_NOT_FOUND, ErrorCode.USER_AUTH_TOKEN_EXPIRED)` ?